ui/pages/settings/networks-tab/networks-tab.js

```diff
@@ -64,7 +69,7 @@ const NetworksTab = ({ addNewNetwork }) => {
   const networkIsSelected = Boolean(selectedNetwork.rpcUrl);
 
   let networkDefaultedToProvider = false;
-  if (!networkIsSelected) {
+  if (networkIsSelected === false) {
```

Comment:
Similar thing here as above — you don't typically need to check that this is exactly `false` unless there is a specific reason for doing so — `!networkIsSelected` is more idiomatic.

``` suggestion
  if (!networkIsSelected) {
```

Reply:
@mcmire You are write, I commited your suggestion. I remembered that I have problem with `lint` when I created this PR (a long time ago), and because of that I put `=== false` or `=== true`. But now it is working perfect, `lint` passed! Thank you for this suggestion!